File: src/main/java/com/amazonaws/kinesisvideo/signaling/Signaling.java
package com.amazonaws.kinesisvideo.signaling;


import com.amazonaws.kinesisvideo.signaling.model.Event;
import com.amazonaws.kinesisvideo.signaling.model.Message;


public interface Signaling {

    void onSdpOffer(Event event);

    void onSdpAnswer(Event event);

    void onIceCandidate(Event event);

    void onError(Event event);
}

File: app/src/main/java/com/amazonaws/kinesisvideo/signaling/Signaling.kt
package com.amazonaws.kinesisvideo.signaling

import com.amazonaws.kinesisvideo.signaling.model.Event

interface Signaling {
    fun onSdpOffer(event: Event?)
    fun onSdpAnswer(event: Event?)
    fun onIceCandidate(event: Event?)
    fun onError(event: Event?)
    fun onException(e: Exception?)
}